ducktape/services/service.py

```diff
@@ -173,12 +176,33 @@ def start_node(self, node):
         """Start service process(es) on the given node."""
         raise NotImplementedError("%s: subclasses must implement start_node." % self.who_am_i())
 
-    def wait(self):
+    def wait(self, timeout_sec=600):
```

Comment:
After some offline discussion, we decided not to do this since it would suggest a timeout for the entire run() operation, which might require timeouts in start() and stop() as well.